### PR TITLE
Adding game ownership

### PR DESCRIPTION
**Pull request by BryceT516**
Stacks on top of PR#9
Adding ownership to the games. This is a belongs_to reference to a player.

### CODE REVIEW

File: spec/controllers/games_controller_spec.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
   # Game. As you add validations to Game, be sure to
   # adjust the attributes here as well.
   let(:valid_attributes) {
-    attributes_for :game
+    (attributes_for :game).merge({:owner_id => player.id})
```
Comment (bcgoss) [inline spec/controllers/games_controller_spec.rb]: Probably need to modify the factory instead

File: app/views/games/show.html.erb [hunk 2]
```diff
@@ -2,6 +2,10 @@
   <strong>Status:</strong>
   <%= @game.status %>
 </p>
+<p>
+  <strong>Owner:</strong>
+  <%= @game.owner.email %>
```
Comment (bcgoss) [inline app/views/games/show.html.erb]: Suggest Player implement `to_s` such that it returns the email. This line would become `<%= @game.owner %>

File: spec/controllers/games_controller_spec.rb (hunk 1 above)
Comment (BryceT516) [reply spec/controllers/games_controller_spec.rb]: It doesn't seem to work that way. I tried it and all the specs that use valid_attributes complain that there is no owner value.
I found this on the factory_bot readme: "Transient attributes will be ignored within attributes_for and won't be set on the model, even if the attribute exists or you attempt to override it."

File: app/views/games/show.html.erb (hunk 2 above)
Comment (BryceT516) [reply app/views/games/show.html.erb]: I made these changes as you suggested.

**Pull request merged**